### ISSUE TITLE
Add more aspect ratios Presets for Grid View

### ISSUE THREAD

**Issue by darklinkpower** [JosefNemec/Playnite#1468]
**Describe the solution you'd like**
With the upcoming Galaxy 2.0, EGS and the new Steam Library UI, I think it would be a good idea to add the new aspect ratios they are using

They use the following aspect ratios and pixel sizes for covers:

**Steam New  Library UI:**
Resolution: 300x450
Aspect Ratio: 2:3

The Xbox Companion App also uses a 2:3 aspect ratio.

**GOG Galaxy 2.0 and Epic Games Store:** 
Resolution: 342x482
Aspect Ratio: 171:241

As suggested in #1467, a 1:1 ratio may be useful too.

**Comment by rogersachan** [comment]
seconding the 1:1 ratio

**Comment by rogersachan** [comment]
The resolution that Twitch/YouTube/IGDB prefer from submitters is 300x400 (aspect ratio of 3:4). I think this should be added as well.

**Comment by darklinkpower** [comment]
@JosefNemec I submitted two PR to add those aspect ratios.
Presets: #1469
Strings (English Only): #1470


**Added aspect ratios:** 


`Generic Square` **1:1**
`GOG Galaxy 2.0 and Epic Games Store` **171:241**
`IGDB and Twitch` **3:4** 

@rogersachan Thanks for the suggestion. I asked to the Igdb team and they say that in fact, they aim for that aspect ratio (even though most of the are inconsistent currently). I didn't mention Youtube in this string because it's not a game or metadata source

`Steam New UI` **2:3**

I didn't add the strings to all languages because I figured you must have some way to add them faster rather than me adding them manually to each language, if not tell me. The added strings are this ones:
```
LOCSettingsCovertAspectGenericSquare
LOCSettingsCovertAspectNewGalaxyEpicGames
LOCSettingsCovertAspectIgdbTwitch
LOCSettingsCovertAspectNewSteamUI
```
